trade-finance-manager-ui/templates/utilisation-reports/_macros/keying-sheet-table.njk

```diff
@@ -0,0 +1,43 @@
+{% import "../../_macros/select-all-table-cell-checkbox.njk" as selectAllTableCellCheckbox %}
+{% import "./keying-sheet-table-row.njk" as keyingSheetTableRow %}
+{% macro render(params) %}
+  {% set reportId = params.reportId %}
+  {% set keyingSheet = params.keyingSheet %}
+  {% set userCanEdit = params.userCanEdit %}
+  <table class="govuk-table" data-module="moj-sortable-table" data-cy="keying-sheet-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header" aria-sort="none" rowspan="2">Status</th>
+        <th scope="col" class="govuk-table__header" aria-sort="ascending" rowspan="2">Facility ID</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none" rowspan="2">Exporter</th>
+        <th scope="col" class="govuk-table__header" rowspan="2">Date</th>
+        <th scope="col" class="govuk-table__header govuk-table__header--numeric" rowspan="2">Fee payment</th>
+        <th scope="col" class="govuk-table__header" rowspan="2">Base currency</th>
+        <th scope="col" class="govuk-table__header" colspan="2">Fixed fee adjustment</th>
+        <th scope="col" class="govuk-table__header" colspan="2">Premium accrual balance adjustment</th>
+        <th scope="col" class="govuk-table__header" colspan="2">Principal balance adjustment</th>
```

Comment:
I think these should have `scope="colgroup"` see here: https://www.w3.org/WAI/EO/Drafts/tutorials/tables/scope/#:~:text=Scope%20is%20the%20attribute%20used,cells%20in%20all%20three%20columns.

Reply:
Interesting - I didn't know those existed. Thanks!